### ISSUE TITLE
Bonus Drive formatted New3DS's screen blinked

### ISSUE THREAD

**Issue by MelonGx** [d0k3/GodMode9#79]
I don't know if it's a BonusDrive issue or a Luma3DS issue.
Unchecking all boxes of Luma3DS setup can reduce the blink but cannot remove it.

To fix this problem, we need to either uninstall B9S or uninstall BonusDrive.
(Both of them are confirmed myself.)

But I need to restore a 1.8GB full NAND backup in order to uninstall BonusDrive.
If I only had 1.2GB minimized NAND backup, then the BonusDrive would be still there after restored it.

**Comment by d0k3** [comment]
The bonus drive can't cause this, absolutely impossible. This is a N3DS, right? Can you give me a more detailed description of exactly what you did?

**Comment by MelonGx** [comment]
Minsize never wipes out Bonus Drive.
If it could, I would have already resolved my last 1.84GB N3DSLL's blinking problem.
I can only uncheck every Luma3DS setting choice to reduce the blinking.

I have 3 1.84GB Toshiba NAND N3DS(LL)s currently (one of them is for sale).
Two of them have 1.84GB backup.
One of them have only Minsize backup but available to be uninstall B9S.

**Comment by d0k3** [comment]
The bonus drive has nothing to do with this. I've got one, too. There was another issue on that NAND of yours.

**Comment by arbingordon** [comment]
I set up a bonus drive since it first was available in GodMode9... never had any issues in 4+ nand backups/restorations, you have another issue.

It sounds like your screen is acting up, so I recommend opening it up and pulling the battery out for a bit and then seeing if you still have the issue.  I had a screen issue before on my N3DS, where there was a weird flickering issue, which was resolved by doing the former.

**Comment by MelonGx** [comment]
@arbingordon It doesn't work. Still blinking on that N3DSLL.
@d0k3 I'll change the CFW into either ReiNAND or CakesFW. If the blinking still happens on both of them, I can only treat it as either BonusDrive or BonusDrive on specific region's issue. BTW, all of my consoles are JPN region. No USA/EUR/KOR/TWN/CHN.

**Comment by arbingordon** [comment]
How long did you leave the battery out?

**Comment by MelonGx** [comment]
@arbingordon 15 minutes.

**Comment by Hikari-chin** [comment]
This **does** seem to be a legitimate issue. I'm experiencing it as well. I triggered it like this:

1. Mounted a title from A: 
2. Copied that title's .code to my SD card
3. Rebooted.
4. Afterwards, the screens began blinking and Luma would not progress past the splash screen; the screens would never turn back on afterwards.
5. I saw this issue, formatted the bonus drive, and now Luma boots but the screens are still blinking.

Solutions I've attempted:
1. Left the system off for several minutes.
2. Pulled the system's battery and waited several minutes.
3. Pulled the system's battery and discharged any remaining current by holding the power button, then waited a few several minutes.

Notes: Intensity of the blinking seems to go down the longer the system is on, but never disappears entirely. Additionally, there's a whitish border around the edges of the screen.

**Comment by d0k3** [comment]
Okay, this seems to be a legit issue. @MelonGx - we need to get behind the real issue. The bonus drive itself resides in unused space and can't have any effect on the system.

Where you using the latest release or the latest hourly of GodMode9? What branch?

Now, do this: Do what you have to do to get this issue on one of your consoles, but make (1) a minsize NAND backup, (2) a CTRNAND backup before. After you got the issue, do the same, so that you now have pre- and post-issue backups of both.

Now, restore the minsize backup from before **twice**, without rebooting in between. Does that fix it?

Also compare those dumps you made. Where's (which offset) the first difference for the two minsize NAND dumps? Which files are different in the CTRNANDs? You'll need to mount both on PC (via OSFMount) and compare via WinMerge.

**Comment by d0k3** [comment]
@MelonGx - jus twriting this so you can continue your experiments. if you're really, really convinced that the bonus drive is responsible, copy the bonus.bin from S:/ brfore setting it up. Set it up, reboot to check the issue is there. Then boot back to GM9 and write the bonus.bin back to remove the bonus drive. Reboot again. Now, is it gone?

**Comment by MelonGx** [comment]
I'm tired that I will do that 1 day later.

**Comment by MelonGx** [comment]
Part 1
1) Make a full 1.84GB NAND backup
2) Make a 1.21GB NAND_minsize backup
3) Make a CTRNAND_full backup
4) Make a bonus_0x26C000 backup

Part 2
5) Install Bonus Drive
6) Make a 1.21GB NAND_minsize2 backup
7) Make a CTRNAND_full2 backup
8) Restore NAND_minsize twice without rebooting in between and check the

Part 3
9) Restore the full 1.84GB NAND backup (or anything can remove Bonus Drive)
10) Install Bonus Drive
11) Restore bonus_0x26C000 backup

Awaiting for the processes to be done.

**Comment by d0k3** [comment]
Okay, and some more... (1) you can also compile GM9 from up to date master (hourlies page borked atm) also (2) does the problem happen after a poweroff, too? Or only after a reboot?

**Comment by MelonGx** [comment]
Weird results.....
When performing everything on N3DS-A, the issue didn't happen again on it even got Bonus Drive formatted.
When restoring N3DS-A's bonus_0x26C000 into N3DSLL, the issue on N3DSLL didn't vanish.

**Comment by MelonGx** [comment]
Just ran them a day.
N3DS-A has no symptoms of this issue even with Bonus Drive installed.
N3DSLL still blinks even with Bonus Drive uninstalled - Only disabling Luma's choices can reduce the blinking.
So I think this is probably a hardware-based legit issue currently.
Temporary closing this issue.

**Comment by MelonGx** [comment]
Try exchanging lower screen on N3DSLL and the blinking never happens anymore even with enabling multiple Luma options.
Seems that it's a hardware issue more than others.

**Comment by d0k3** [comment]
Exchanging the screens? You mean you had a broken screen on there? Reopening this, just to make sure I know everything I need to know.

**Comment by MelonGx** [comment]
@d0k3 It's not broken but possibly in a not-so-good condition.
Now the blinking on that N3DSLL is mainly caused by enabling Luma3DS Splash, regardless of installing Bonus Drive.

**Comment by MelonGx** [comment]
@d0k3 It seems to be an IPS-lower-screen x enabling Luma3DS Splash issue now.
Just exploited a Double-IPS N3DSLL with 1.21GB Samsung NAND - which cannot establish a Bonus Drive.
The lower screen blinks after B9S installed & Luma3DS Splash enabled.

**Comment by d0k3** [comment]
Okay, thank you. Can you close and reopen this issue on the Luma 3DS repo? Also recheck if it still happens with the most recent releases of boot9strap (v1.1) and Luma.

**Comment by MelonGx** [comment]
@d0k3 Since I checked that not only Luma3DS but also ReiNAND & Corbenik had this problem either, I put it into boot9strap's issue list.

**Comment by Hikari-chin** [comment]
Just had this reoccur for me after launching and rebooting from GM9. Seems really random, wish we could get to the bottom of this.

**Comment by WubTheGame** [comment]
.........what's a bonus drive?

**Comment by ihaveamac** [comment]
@WubTheGame README has information on this

**Comment by WubTheGame** [comment]
ok

**Comment by Hikari-chin** [comment]
@WubTheGame Either way, GitHub issues are not the appropriate place to ask questions. You've been doing this pretty consistently across multiple 3DS-related repos, could you stop?

**Comment by WubTheGame** [comment]
sorry... i guess i didn't notice... i'll try to stop.
